libraries/AP_Motors/AP_Motors_Class.h

```diff
@@ -107,10 +107,15 @@ class AP_Motors {
     //
     // set_voltage - set voltage to be used for output scaling
     void                set_voltage(float volts){ _batt_voltage = volts; }
+    void                set_voltage_resting_estimate(float volts) { _batt_voltage_resting_estimate = volts; }
 
     // set_current - set current to be used for output scaling
     void                set_current(float current){ _batt_current = current; }
 
+    // get and set battery resistance estimate
+    float               get_batt_resistance() const { return _batt_resistance; }
+    void                set_resistance(float resistance){ _batt_resistance = resistance; }
```

Comment:
the motors lib actually uses the resistance in the get_current_limit_max_throttle function.  I definitely agree we've got some duplicate logging of the resistance but I think for support purposes, @lthall would like the resistance to be logged at the same time as the other values used to limit the motor output...